roles/accprovision/manifests/net-operator-controller-deployment.yaml

```diff
@@ -0,0 +1,410 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: net-operator-controller
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: net-operator-controller
```

Comment:
Lets keep the namespace name as "net-operator-controller" but change all other occurrences below to network-operator-fabric-controller.